src/ReactiveUI/CommandBinding.cs

```diff
@@ -30,7 +33,7 @@ static CommandBinder()
         /// <param name="view">The View</param>
         /// <param name="viewModel">The View model</param>
         /// <param name="controlName">The name of the control on the view</param>
```

Comment:
In a separate commit/PR I'll have a look at lining up controlProperty/controlName.

As a general stylistic point, are we aiming for extension methods to be on a parameter called `This`, `@this` or on something like these ones like `view`?